### PR TITLE
Update Magento_Authorizenet ReadMe

### PR DESCRIPTION
**Pull request by diazwatson**
### Description (*)

Updates ReadMe for the Magento_Authorizenet module.

### Fixed Issues (if relevant)

1. magento/devdocs https://github.com/magento/devdocs/issues/4451: README content for modules - work in M2 Code

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Authorizenet/README.md [hunk 1]
```diff
@@ -1 +1,42 @@
-The Magento_Authorizenet module implements the integration with the Authorize.Net payment gateway and makes the latter available as a payment method in Magento.
+# Magento_Authorizenet module
+
+The Magento_Authorizenet module is a part of the staging functionality in Magento EE. The module adds the “Configurations” tab and the configuration wizard to the Schedule Update form of a product. You can change the Configurable Product attributes in campaigns. These updates are shown on the campaign dashboard.
```
Comment (jeff-matthews) [inline app/code/Magento/Authorizenet/README.md]:
```suggestion
The Magento_Authorizenet module is a part of the staging functionality in Magento Commerce. The module adds the “Configurations” tab and the configuration wizard to the Schedule Update form of a product. You can change the Configurable Product attributes in campaigns. These updates are shown on the campaign dashboard.
```

File: app/code/Magento/Authorizenet/README.md [hunk 2]
```diff
@@ -1 +1,42 @@
...
+   - `result` is a data object (`\Magento\Framework\DataObject` class).
+   - `action` is a controller object (`\Magento\Authorizenet\Controller\Directpost\Payment\Place`).
+ 
+ - `order_cancel_after` event in the `\Magento\Authorizenet\Model\Directpost::declineOrder()` method. Parameters:
+   - `order` is an order object (`\Magento\Sales\Model\Order` class).
+   
+
+This module observes the following events:
+
+ - `checkout_submit_all_after` event in `Magento\Authorizenet\Observer\SaveOrderAfterSubmitObserver` file.
```
Comment (jeff-matthews) [inline app/code/Magento/Authorizenet/README.md]:
```suggestion
 - `checkout_submit_all_after` event in the `Magento\Authorizenet\Observer\SaveOrderAfterSubmitObserver` file.
```

File: app/code/Magento/Authorizenet/README.md [hunk 3]
```diff
@@ -1 +1,42 @@
...
+ - `checkout_directpost_placeOrder` event in `Magento\Authorizenet\Observer\AddFieldsToResponseObserver` file.
```
Comment (jeff-matthews) [inline app/code/Magento/Authorizenet/README.md]:
```suggestion
 - `checkout_directpost_placeOrder` event in the `Magento\Authorizenet\Observer\AddFieldsToResponseObserver` file.
```

File: app/code/Magento/Authorizenet/README.md [hunk 4]
```diff
@@ -1 +1,42 @@
...
+
+For information about an event in Magento 2, see [Events and observers](http://devdocs.magento.com/guides/v2.3/extension-dev-guide/events-and-observers.html#events).
```
Comment (jeff-matthews) [inline app/code/Magento/Authorizenet/README.md]:
```suggestion
For information about events in Magento 2, see [Events and observers](http://devdocs.magento.com/guides/v2.3/extension-dev-guide/events-and-observers.html#events).
```

**Review by jeff-matthews — CHANGES_REQUESTED** [review]

**Review by jeff-matthews — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @jeff-matthews, thank you for the review.
[ENGCOM-5706](https://jira.corp.magento.com/browse/ENGCOM-5706) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
Hi @diazwatson I think module description after **# Magento_Authorizenet module** header is not correct.
Could you take a look?

**Comment by diazwatson** [comment]
@engcom-Delta many thanks for catching this 👍

**Comment by ihor-sviziev** [comment]
Hi @jeff-matthews,
Could you review latest changes that were made by @diazwatson?

**Review by jeff-matthews — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @jeff-matthews, thank you for the review.
[ENGCOM-5706](https://jira.corp.magento.com/browse/ENGCOM-5706) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

**Pull request merged**